docs/guides/getting-started-orchestrate-human-tasks.md

```diff
@@ -206,29 +249,54 @@ If the properties panel for your task doesn't open automatically, navigate to **
 
-### Deploy and test run
+1. Click the **Play** tab to enter Play mode.
+1. Once the Play environment is ready, click **Start a process instance** to start testing your process.
+1. Start by activating a process instance. Click **Play** on the canvas above the process start event.
+   <img src={PlayButtonImg} style={{width: 300}} alt="Start playing the process instance" />
+1. The token moves to the user task, stops and waits until the user task is completed. Click **Open Task Form** to open the form.
+   <img src={PlayOpenFormImg} style={{width: 300}} alt="Complete the form and continue" />
 
-<Tabs groupId="install" className="tabs-hidden">
-<TabItem value="saas">
+   <p>In the form, choose the <code>Chicken</code> option, and click <strong>Complete</strong>.</p>
 
-1. Click **Deploy** to deploy the process to your cluster. If you have not yet created a cluster, clicking **Deploy** will take you to Console to [create a cluster](create-cluster.md) first.
-2. After you deploy your process, it can be executed on the cluster. There are multiple ways to run a process. This time, click **Run** in Modeler for a test run.
+1. The token moves through the exclusive gateway (also called the XOR gateway), and is used to model the decision in the process.
+   <img src={PlayChickenImg} style={{width: 800}} alt="Completing the Play process" />
+
+   <p>When the execution arrives at this gateway, all outgoing sequence flows are evaluated in the order in which they have been defined. The sequence flow which condition evaluates to ‘true’ is selected for continuing the process.</p>
+   <p>In this example, the token moved through the gateway and (according to the conditional expressions outlined earlier) to the selected dinner based on the <code>Decide what's for dinner</code> user task you completed.</p><p>As you chose <code>Chicken</code>, the token moved through to <code>Prepare chicken</code> and successfully completed. If you had selected <code>Salad</code>, the token would have moved through to <code>Prepare salad</code> instead.</p>
+
+1. Click **Rewind** on the canvas and select the `Salad` option to test this flow also works correctly.
+
+:::note
+
+As well as using Play mode to quickly validate and run your process in development, you can also:
+
+- Deploy the process to a [cluster](/components/concepts/clusters.md) in other environments such as testing, staging, and production. After you deploy your process, it can be run on the cluster.
+- Run and complete the user task in [Tasklist](/components/tasklist/introduction-to-tasklist.md). Applications such as Tasklist can be used by humans to complete tasks. As well as using Play mode and Tasklist to run a process, you can call the process via the API or an inbound trigger. Read more about [run options](/components/modeler/web-modeler/run-or-publish-your-process.md).
```

Comment:
You can follow a link to Operate...
![image](https://github.com/user-attachments/assets/42c511d3-7748-419a-859c-ac579a640525)
and then a link to Tasklist if you have an open user task...
![image](https://github.com/user-attachments/assets/748418bc-906f-4d0b-82e2-5f8cf6ace7fb)

Or you can open Tasklist and it should show the open task directly